### DISCUSSION TITLE
Need to change who is a member on our Hobby account

### DISCUSSION THREAD

**Discussion by glenna** [vercel/community#2956] [Help]
### Summary

Hi, we are using a Hobby Account for some internal tooling. Since it is a Hobby account, there is only one member (me), but I need to transfer the ownership to someone else. Is it possible to do it without a temporary account upgrade?

### Steps to Reproduce

Just need help with account info

**Comment by amyegan** [comment]
Hi @glenna. You would need to put both Hobby accounts on a Pro team, and then you can transfer projects to the appropriate owner. Hobby accounts can't connect directly to each other so there is currently no way to transfer without a Team to connect them.
